src/render/src/doc/terminology.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 The Gfx-rs Developers.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// This module contains documentation for the technical terms.
+// The goal of this part of the documentation is to bridge the gaps between
+// the terminology employed by gfx and other graphics APIs.
+
+
+//! # Terminology
+//!
+//! ## Buffer
+//!
+//! TODO!
+//!
+//! ## Handle
+//!
+//! Handles are the mechanism primarily gfx uses to refer to most GPU resources.
+//! Most handle types use an atomically reference counted pointer internally to automatically
+//! deal with memory management. This means that when using the gfx's high level API, there is
+//! no need to explicitly manage the lifetime and destroy handles.
+//!
+//! In OpenGL, handles are simple integers (contrary to gfx's high level API, OpenGL does not
+//! provide any automatic memory management).
+//!
+//! Note: The comments about memory management above apply to the high level API, but not the
+//! low level one (ll).
+//!
+//! ## Slice
+//!
+//! A `Slice` in gfx describes for a givien draw call the range of vertices to process, their order
+//! and the number of instances.
+//! The slice is how index buffers are specfied in a draw call.
+//!
+//! Not to confuse with the term slice as employed more broadly in rust (`&[T]`) which is a view into
```

Comment:
I agree. It confused me until I looked into its code, and the notion of slice is so central in rust that I think it would be good to avoid overloading it.